### PR TITLE
GitHub actions / Test CI / Add PostgreSQL tests

### PR DESCRIPTION
**Pull request by jenovateurs**
Hello team!

* Type: bug fix | code quality | documentation
* Link to issue: #1420 

**In raising this pull request, I confirm the following (please check boxes):**

- [x] I have read and understood the [Contributing Guidelines][:contrib:]
- [x] I have checked that another pull request for this purpose does not exist
- [x] I wrote some tests for this PR

A lot of work has been done.
GitHub Actions works on my fork but I didn't try with Travis. 
I think, I will get some errors that I will solve.
I added two groups one for mysql, second for pgsql like we can found in cphalcon project.
About CODECOV, I don't have the CODECOV_TOKEN, if someone can add it on the project ;-).

Thanks

[:contrib:]: https://github.com/phalcon/phalcon-devtools/blob/master/CONTRIBUTING.md

### CODE REVIEW

**Comment by jenovateurs** [comment]
Still in progress...

**Comment by jenovateurs** [comment]
I'm done changing things in this PR. You can find the result of GitHub CI here : 
https://github.com/les-enovateurs/phalcon-devtools/runs/565050607?check_suite_focus=true
About PR Quality, is it possible to ignore it ?
There are old problems like $I is too short, ...

File: tests/_ci/setup_dbs_github.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# LICENSE.txt file that was distributed with this source code.
+
+CURRENT_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+BUILD_DIR="${BUILD_DIR:-"$( dirname "$( dirname "$CURRENT_DIR" ) ")"}"
+echo -e "Create PostgreSQL database..."
+psql -U postgres devtools -p "$POSTGRES_DB_PORT" -h localhost -q -f "${BUILD_DIR}/tests/_data/schemas/postgresql/dump.sql"
+echo -e "Done\n"
+
+echo -e "Create MySQL database..."
+cat "${BUILD_DIR}/tests/_data/schemas/mysql/dump.sql" | mysql --host=127.0.0.1 --user=root --password=root --port="$MYSQL_DB_PORT" --database devtools
```
Comment (Jeckerson) [inline tests/_ci/setup_dbs_github.sh:17]: I suggest to create also MYSQL_DB_PASSWORD env variable in `.env.test` and pass it to here.

File: tests/_support/Helper/Acceptance.php [hunk 2]
```diff
@@ -24,6 +24,17 @@ public function _initialize()
 
     public function _before(TestInterface $test)
     {
+        $driver = $test->getMetadata()->getCurrent('env');
+        if (!empty($driver)) {
+            $this->driver = $driver;
+        }
+
+        if (true === in_array($driver, [ 'mysql', 'pgsql' ])) {
```
Comment (Jeckerson) [inline tests/_support/Helper/Acceptance.php:32]:
```suggestion
        if (true === in_array($driver, ['mysql', 'pgsql'])) {
```

File: tests/_data/acceptance/pgsql/config.php [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+defined('APP_PATH') || define('APP_PATH', BASE_PATH . '/app');
+
+return new \Phalcon\Config([
+    'database' => [
+        'adapter'  => 'Postgresql',
+        'host'     => 'localhost',
+        'username' => 'postgres',
+        'password' => '',
+        'dbname'   => 'devtools',
+        'port'     => getenv('POSTGRES_DB_PORT')
```
Comment (Jeckerson) [inline tests/_data/acceptance/pgsql/config.php:17]:
```suggestion
        'port'     => getenv('POSTGRES_DB_PORT'),
```

Comment (jenovateurs) [reply tests/_data/acceptance/pgsql/config.php]: Updated ;-)

**Comment by jenovateurs** [comment]
I think that good . Need second review;
What do you think @Jeckerson

File: tests/_ci/setup_dbs_github.sh (hunk 1 above)
Comment (jenovateurs) [reply tests/_ci/setup_dbs_github.sh:17]: Updated :-) Thanks for your review.

**Pull request merged**